src/test/resources/org/jabref/logic/exporter/ModsExportFormatTestAllFields.xml

```diff
@@ -11,7 +11,7 @@
             <mods:namePart type="given">know</mods:namePart>
         </mods:name>
         <mods:name type="personal">
-            <mods:namePart type="family">Realy</mods:namePart>
+            <mods:namePart type="family"> Realy</mods:namePart>
```

Comment:
Why are there spaces before the value here and in the other xml?

Reply:
See above, this is exactly the output of the exporter. I didn't change anything in the logic but strengthened the file comparison in the tests. (Probably needs to be done for the other exporters as well)